opensrp-chw/build.gradle

```diff
@@ -374,7 +374,7 @@ android {
 }
 
 dependencies {
-    implementation('org.smartregister:opensrp-client-chw-core:2.1.3-SNAPSHOT-KK-5@aar') {
+    api(project(':opensrp-chw-core')) {
```

Comment:
Is this still the reference?

Reply:
I'll update the chw-core snapshot version with the changes required for HF referrals then push the commit with the dependency version